core/server/apps/default-cards/tests/hr_spec.js

```diff
@@ -0,0 +1,18 @@
+var should  = require('should'),
+    card    = require('../cards/hr'),
+    SimpleDom   = require('simple-dom'),
+    opts;
+
+should = should;
+describe('HR card', function () {
+    it('generates a horizontal rule', function () {
+        opts = {
+            env: {
+                dom: new SimpleDom.Document()
+            }
+        };
+
+        var serializer = new SimpleDom.HTMLSerializer([]);
+        serializer.serialize(card.render(opts)).should.match('<hr></hr>');
```

Comment:
I don't know if it's worth putting a PR through to SimpleDom so that void tags are rendered properly or not. ^ should obviously be <hr>, also see <br> and <img> below.